src/AppBundle/Entity/Comment.php

```diff
@@ -16,6 +16,7 @@
 
 /**
  * @ORM\Entity
+ * @ORM\Table(name="symfony_demo_comment")
```

Comment:
@voronkovich I didn't know about this. Thanks for sharing that link.

I don't know if others will agree, but I think that solution is too advanced even for experienced developers.